script/output/scroll_sepolia/v1-deployment.json

```diff
@@ -0,0 +1,16 @@
+{
+  "addresses": {
+    "queryClient": "0x0000000000000000000000000000000000000000",
```

Comment:
THis is not normal

Reply:
@nikkolasg This is fine, it just means no contract is deployed

Reply:
Yes, super normal when we only deploy the registry. I can deploy the query client also, but I was under the impression we didn't want to spend more time on Scroll deployment than necessary.